docs/README.md

```diff
@@ -22,6 +22,11 @@ Run:
 
 To use an API key, put it in `server/secrets.py` or as an environment variable `MBTA_V3_API_KEY`
 
+On Windows:
+Before running `npm install`,
+- Ensure that `python3` in the `start-python`,`create-history-db`, and `update-history-db` is updated to `python`.
```

Comment:
Since we use pipenv, maybe we can just change `package.json` to use `python` all of the time?

Reply:
Either way, I'd prefer we don't have a solution that requires changing git-tracked files for dev work